calyx/src/backend/rtl/gen.rs

```diff
@@ -23,6 +23,7 @@ impl Backend for RtlBackend {
                         }
                     }
                 }
+                Control::Enable { .. } => (),
```

Comment:
Is this just to accept the case when we have no top-level `seq` and just a singular `enable`?

Reply:
Yes. I saw issue #65 and changed it